superset-frontend/src/views/CRUD/dataset/DatasetList.tsx

```diff
@@ -448,7 +448,11 @@ const DatasetList: FunctionComponent<DatasetListProps> = ({
         label: t('Datasets'),
         url: '/tablemodelview/list/',
       },
-      { name: 'Databases', label: t('Databases'), url: '/databaseview/list/' },
+      {
+        name: 'Datasources',
+        label: t('Datasources'),
+        url: '/databaseview/list/',
```

Comment:
I vote for moving to `/database/list/`. The "view" suffix is a FAB artifact from what the ModelView class was class.